application.go

```diff
@@ -122,10 +122,11 @@ func (r *Application) AllTaskRunning() bool {
 	if r.Tasks == nil {
 		return false
 	}
-	if r.TasksRunning == r.Instances {
-		return true
+	if r.HasHealthChecks() {
+		return r.TasksRunning == r.Instances && r.TasksHealthy == r.Instances
+	} else {
+		return r.TasksRunning == r.Instances
```

Comment:
Hi @timoreimann 
not every app has healthchecks. Starting a simple app shows TasksHealthy in 0. That is why I added the condition with HasHealthChecks.